django/contrib/auth/views.py

```diff
@@ -224,6 +224,9 @@ def password_reset(request,
                    extra_context=None,
                    html_email_template_name=None,
                    extra_email_context=None):
+    warnings.warn("Function-based views for password reset are deprecated. "
```

Comment:
could you follow the message style  of login/logout? e.g. 'The logout() view is superseded by the class-based LogoutView().'